### ISSUE TITLE
Can you provide more flexible way (regex) to exclude files/directories from sync?

### ISSUE THREAD

**Issue by abougouffa** [meganz/MEGAsync#505]
Hi,

I would like to suggest adding the possibility to exclude files/directories based on regular expressions. For example, in my case I want to exclude **all directories containing a file named `CACHEDIR.TAG`** (rustc cache dir).

By adding regular expressions support, users will be able to customize thier exclusion rules much efficiently.

Thank you,

**Comment by mattw-mega** [comment]
Yes we can!  Hi @abougouffa , you may like to know we have a big update coming for MEGAsync, which includes exclusions by .megaignore file, which can use regex expressions.  It would be great to know if it works for you. You can read about it and perhaps help test it here: https://github.com/meganz/MEGAsync/issues/770 thanks
